harvester_e2e_tests/integration/test_images.py

```diff
@@ -0,0 +1,212 @@
+
+from harvester_e2e_tests.apis import test_images, test_volumes
+
+from time import sleep
+
+import pytest
+import os
+
+
+pytest_plugins = [
+    "harvester_e2e_tests.fixtures.api_client",
+    'harvester_e2e_tests.fixtures.image',
+    'harvester_e2e_tests.fixtures.vm',
+]
+
+
+@pytest.mark.images_p1
+@pytest.mark.dependency(name="create_image_from_volume")
+def test_create_image_from_volume(api_client):
+    """
+    Test create image from volume
+
+    Steps:
+    1. Create a volume "test-volume" in Volumes page
+    2. Export the volume to image "export-image"
+    3. Check the image "export-image" exists
+    4. Cleanup image "export-image" on Images page
+    5. Cleanup volume "test-volume" on Volumes page
+    """
+
+    volume_name = 'test-volume'
+    export_name = 'export-image'
```

Comment:
`volume_name` and `export_name` should use unique name instead of the **hard-coded** one.
we already have `unique_name` fixture for you to make it possible.

Reply:
Use `unique_name` for `volume_name` and `export_name`